documentation/modules/rn-2.2.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-22_{context}"]
+= {project-full} 2.2
+
+// [id="technical-changes-22_{context}"]
+// == Technical changes
+//
+// .TBD
+//
+// TBD
+//
+// [id="new-features-and-enhancements-22_{context}"]
+// == New features and enhancements
+//
+// This release adds the following features and improvements.
+//
+// .TBD
+//
+// TBD
+
+[id="known-issues-22_{context}"]
+== Known issues
+
+.QEMU guest agent is not installed on migrated VMs
+
+The QEMU guest agent is not installed on migrated {rhv-full} VMs. Workaround: Install the QEMU guest agent with a post-migration hook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[*BZ#2018062*])
```

Comment:
QEMU guest agent installation is missing for both RHV & VMware.

Reply:
OK. I'll remove the RHV reference.